File: python/ex_00_connectivity.py
from db_connection import get_session

if __name__ == '__main__':
    session = get_session()
    #
    rows = session.execute("SELECT key, cluster_name, data_center FROM system.local;")
    local = rows.one()
    if local:
        print("    ** Connected to cluster '%s' at data center '%s' **" % (
            local.cluster_name,
            local.data_center,
        ))
    else:
        print("Error: could not read 'system.local' table!")

File: javascript/ex_00_connectivity.js
const {getSession} = require('./db_connection');

async function main() {

  const session = getSession();
  await session.connect();

  session.execute("SELECT key, cluster_name, data_center FROM system.local;")
    .then(result => {
      const local = result.first();
      console.log(`    ** Connected to cluster '${local['cluster_name']}' at data center '${local['data_center']}' **`);
      session.shutdown().then( () => console.log('Session closed.'));
    });

}

main();
